guides/source/getting_started.md

```diff
@@ -605,9 +597,12 @@ class ArticlesController < ApplicationController
 end
 ```
 
-If you re-submit the form now, you'll see another familiar error: a template is
-missing. That's ok, we can ignore that for now. What the `create` action should
-be doing is saving our new article to the database.
+If you re-submit the form now, you may not see any change on the page. Don't worry!
+This is because Rails by default returns `204 No Content` response for an action if
+we don't specify how the response should be. In this case we just added the `create` action
+but didn't specify anything about how the response should be so Rails is returning with
+`204 No Content` response. In this case, what the create action should be doing is saving
```

Comment:
Repetition of text, I would keep the original content- "What the `create` action should
 be doing is saving our new article to the database."